pkg/rpm.bzl

```diff
@@ -233,7 +233,10 @@ Args:
   changelog: A changelog file to include. This will not be written to the spec
     file, which should only list changes to the packaging, not the software itself.
   source_date_epoch: Value to export as SOURCE_DATE_EPOCH to facilitate reproducible
+    timestamps.  Implicitly sets the `%clamp_mtime_to_source_date_epoch` in the
+    subordinate call to `rpmbuild` to facilitate more consistent in-RPM file
     timestamps.
-  source_date_epoch_file: File containing the SOURCE_DATE_EPOCH value
+  source_date_epoch_file: File containing the SOURCE_DATE_EPOCH value.  Sets
```

Comment:
Is a file the way most people use SOURCE_DATE_EPOCH?
or should this, instead, to the glue to pass that from the caller environment?
or is this fine for now and you can do the other later?

Reply:
I legit don't know.  We don't use it anywhere that I know of.

This is primarily to establish parity with changes added during #113.

The most correct way to do this off the top of my head would to pass this in via [`--workspace_status_command`](https://docs.bazel.build/versions/master/user-manual.html#flag--workspace_status_command).

Reply:
OK. I was just asking. I don't know either.